lib/Backend/GlobOpt.cpp

```diff
@@ -3489,7 +3489,11 @@ GlobOpt::OptSrc(IR::Opnd *opnd, IR::Instr * *pInstr, Value **indirIndexValRef, I
 
     if (opnd->IsSymOpnd() && opnd->AsSymOpnd()->IsPropertySymOpnd())
     {
-        TryOptimizeInstrWithFixedDataProperty(&instr);
+        if (instr->m_opcode != Js::OpCode::LdSuperFld) // TODO: should profile LdSuperFld?
```

Comment:
Yes, I would prefer to profile LdSuperFld. It seems best to let it participate in this optimization. If there's a reason we can't do that, I'd prefer to use some kind of general check on the opcode attributes rather than special-casing for this opcode.